oops/backplane/unittester.py

```diff
@@ -1,289 +1,60 @@
 ################################################################################
 # oops/backplane/unittester.py
 ################################################################################
+#
+# Usage:
+#   python unittester.py [--output dir] [--no_output] [--exercises_only]
```

Comment:
Unix standard is to have all dashes in an option, not a mixture of dashes and underscores